src/native.jl

```diff
@@ -7,6 +7,7 @@ export NativeCompilerTarget
 Base.@kwdef struct NativeCompilerTarget <: AbstractCompilerTarget
     cpu::String=(LLVM.version() < v"8") ? "" : unsafe_string(LLVM.API.LLVMGetHostCPUName())
     features::String=(LLVM.version() < v"8") ? "" : unsafe_string(LLVM.API.LLVMGetHostCPUFeatures())
+    always_inline::Bool=false
```

Comment:
Maybe a line of docs to explain this applies to the kernel function itself, which may be useful for inlining the deferred call

Reply:
Yeah, any ideas for better propagating this?